app/models/request.rb

```diff
@@ -120,7 +120,7 @@ def notification_email_address
   end
 
   def submit!
-    send_to_symphony!
+    send_to_symphony_later!
```

Comment:
Can we add a `submit!` method to the `Scan` class that overrides this to do a `send_to_symphony_now!` (given the complication about requiring a browser with webauth credentials to pass through Illiad for scan requests).